packages/web/src/pages/profile-page/MobileServerProfilePage.tsx

```diff
@@ -8,12 +7,9 @@ import { Box } from '@audius/harmony/src/components/layout/Box'
 import { Flex } from '@audius/harmony/src/components/layout/Flex'
 import { Text } from '@audius/harmony/src/components/text'
 
-// import ProfilePageBadge from 'components/user-badges/ProfilePageBadge'
```

Comment:
ProfilePageBadge is commented out below, but can be uncommented now that the profile page state has been added to the server state

Reply:
line 192 here

Reply:
oh right! wil do